docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -250,7 +250,7 @@ The module `org.keycloak:keycloak-model-legacy` module was deprecated in a previ
 = XA Transaction Changes
 
 * The option `transaction-xa-enabled` will default to false, rather than true. If you want XA transaction support you will now need to explicitly set this option to true.
-* XA Transaction recovery support is enabled by default if `transaction-xa-enabled` is true. Transaction logs will be stored at KEYCLOAK_HOME/data/transaction-logs.
+* XA Transaction recovery support is enabled by default. Transaction logs will be stored at KEYCLOAK_HOME/data/transaction-logs.
```

Comment:
```suggestion
* XA Transaction recovery support is enabled by default. Transaction logs will be stored at KEYCLOAK_HOME/data/transaction-logs. If you don't use any XA datasources, you can entirely disable recovery by setting `QUARKUS_TRANSACTION_MANAGER_ENABLE_RECOVERY=false`.
```

Reply:
@zc-devs generally we don't call out the usage of quarkus properties. @vmuzikar do you want to make an exception in this case since it allows the user to maintain the previous behavior?

Reply:
> do you want to make an exception in this case since it allows the user to maintain the previous behavior?

I would not do it. This guide is assumed to be supported while usage of the Quarkus options is not. We'd be instructing the user to do something unsupported in a supported guide. If we'll see there's a bigger need for toggling the recovery, we'll add a dedicated option for it.